zcash_client_backend/CHANGELOG.md

```diff
@@ -232,6 +233,11 @@ and this library adheres to Rust's notion of
   - The fields of `ReceivedSaplingNote` are now private. Use
     `ReceivedSaplingNote::from_parts` for construction instead. Accessor methods
     are provided for each previously public field.
+  - `Recipient` is now polymorphic in the type of the payload for wallet-internal
+    recipients. This simplifies the handling of wallet-internal outputs.
+  - `SentTransactionOutput::from_parts` now takes a `Recipient<Note>`
+  - `SentTransactionOutput::recipient` now returns a `Recipient<Note>`
+  - `OvkPolicy::Custom` now wraps a bare `[u8; 32]` instead of a Sapling `OutgoingViewingKey`.
```

Comment:
If the "document `OvkPolicy::Custom` very carefully" option is taken, this changelog entry needs to be correspondingly expanded.